src/input.rs

```diff
@@ -56,8 +58,9 @@ impl<'a> Input<'a> {
         Self {
             title: title.into(),
             description: String::new(),
-            placeholder: String::new(),
             prompt: "> ".to_string(),
+            placeholder: String::new(),
+            suggestions: vec![],
```

Comment:
A `Vector` is a relatively naive approach, question is if we need support for an insanely large set of suggestions here?

Reply:
Played around with a `Trie` implementation which is 100x faster but makes case insensistive matching impossible. Also with for example 5k name suggestions, names overlap that much that the suggestion would only suggest the next character.